cmake/OpenCVGenConfig.cmake

```diff
@@ -41,7 +41,7 @@ foreach(m ${OPENCV_MODULES_BUILD})
   endif()
 endforeach()
 
-export(TARGETS ${OpenCVModules_TARGETS} FILE "${CMAKE_BINARY_DIR}/OpenCVModules.cmake")
+export(TARGETS ${OpenCVModules_TARGETS} APPEND FILE "${CMAKE_BINARY_DIR}/OpenCVModules.cmake")
```

Comment:
`APPEND` breaks Android pack build.
Multiple configurations are added to the same file during `build_sdk.py`.

BTW, "install" OpenCVModules doesn't have "APPEND"

Reply:
Looks like we need to remove this file after each platform build. I'll fix the script.

Reply:
`install(EXPORT ...` does not have `APPEND` parameter, I think it just takes the final file and copies it to the destination. `export(TARGETS ...` needs `APPEND` when called from several different places with same filename. Or we can try to use `export(EXPORT ...` variant which will export all targets added to our named export set - it needs to be called only once and does not need `APPEND` argument - I'll try this.